app/src/test/java/com/kickstarter/viewmodels/ProjectViewModelTest.kt

```diff
@@ -974,6 +980,7 @@ class ProjectViewModelTest : KSRobolectricTestCase() {
         this.expandPledgeSheet.assertValue(Pair(true, true))
         this.koalaTest.assertValues("Project Page")
         this.lakeTest.assertValue("Project Page Viewed")
+        this.experimentsTest.assertNoValues()
```

Comment:
Why is `experimentTest` not expected to have a value? Just trying to understand what is being tested. Thanks!

Reply:
This test is testing that when a user clicks the `Back this project` button from the campaign webview, the pledge sheet is expanded on the project page. Since they didn't explicitly click the `Back this project` button on the project page (we're just automatically opening the sheet), we're not tracking that as a click event.